docs/source/dataframe-optimizer.rst

```diff
@@ -38,13 +38,25 @@ The optimizations entail the following steps (this list is not complete):
   ``df.groupby(...).apply(...)`` after a merge operation will not shuffle the data
   again if the groupby happens on the merge columns.
 
+  .. figure:: images/optimizer/avoiding-shuffles.svg
+    :align: center
+
+    Performing two subsequent Joins/Merges on the same join-column(s) will avoid shuffling the
```

Comment:
nit: IMO this makes it clearer that this is still an example and this optimization is not limited to these cases. YMMV, feel free to ignore.

```suggestion
    Similarly, performing two subsequent Joins/Merges on the same join-column(s) will avoid shuffling the
```